docs/rule/no-invalid-meta.md

```diff
@@ -45,6 +47,7 @@ This rule **allows** the following:
 
 ### References
 
+* [HTML meta Tag spec](https://www.w3schools.com/tags/tag_meta.asp)
```

Comment:
Would `HTML meta Tag doc` be more precise?

Reply:
Updated.